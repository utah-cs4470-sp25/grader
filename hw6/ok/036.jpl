struct boolint {
  a: bool
  b: int
}

show boolint{ true, 1 }
