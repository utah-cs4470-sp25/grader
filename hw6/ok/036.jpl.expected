(StructCmd boolint a (BoolType) b (IntType))
(ShowCmd (StructLiteralExpr (StructType boolint) boolint (TrueExpr (BoolType)) (IntExpr (IntType) 1)))
